### PR TITLE
IgnorePlugin: Link at Plugins List Page

### PR DESCRIPTION
**Pull request by welll**
This adds the Link at Plugins List Page

Index Plugins Page:
https://webpack.js.org/plugins/

It complements #1014

### CODE REVIEW

File: content/plugins/index.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ webpack has a rich plugin interface. Most of the features within webpack itself
 |[`DllPlugin`](/plugins/dll-plugin)|Provide means to split bundles in a way that can drastically improve build time performance.|
 |[`ExtractTextWebpackPlugin`](/plugins/extract-text-webpack-plugin)|Extracts Text (CSS) from your bundles into a separate file (app.bundle.css)|
 |[`HtmlWebpackPlugin`](/plugins/html-webpack-plugin)| Simplifies creation of HTML files (`index.html`) to serve your bundles|
+|[`IgnorePlugin`](/plugins/ignore-plugin)| Does not include some modules in your bundles|
```
Comment (skipjack) [inline content/plugins/index.md]: I think "Excludes certain modules from output" or "Excludes certain modules from bundles" would probably sound a bit better.

**Comment by skipjack** [comment]
@welll thanks for making that change, can you add that other email to your account to resolve that cla-bot issue? See the comment above:

> Wellington Soares seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please add the email address used for this commit to your account.

**Comment by welll** [comment]
@skipjack I made some mistakes trying to rebase/squash my commits using only my Github account. I am creating this PR again. Sorry about that.

**Comment by skipjack** [comment]
No worries just ping me when the it's ready 👍

**Pull request closed without merging**